packages/terra-form-select/src/terra-dev-site/test/form-select/ManyOptions.test.jsx

```diff
@@ -29,7 +38,7 @@ const ManyOptionsExample = () => {
       <p>
         Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
       </p>
-      <Select id="frmSelectSingle" placeholder="Select a number" variant="default" maxHeight={200}>
+      <Select id="frmSelectSingle" placeholder="Select a number for Tag" variant="tag" maxHeight={200}>
```

Comment:
Id needs to be changed it still refers to single form-select. I think it can be removed as it is not used in any of the WDIO specs.

Reply:
Reverted the changes made to this file.